bytes/bytes/api/router.py

```diff
@@ -190,6 +191,11 @@ def create_raw(
                 raw_data=RawDataMeta(id=raw_id, boefje_meta=raw_data.boefje_meta, mime_types=raw_data.mime_types),
             )
             event_manager.publish(event)
+        except pika.exceptions.AMQPError as error:
```

Comment:
I'm not sure if this works, because Byte's `RabbitMQEventManager.publish` already handles the `AMQPError` exceptions and doesn't reraise them. That could explain the missing stack traces in the logs. However, while the `RabbitMQEventManager.publish` handles these errors, it could potentially still produce an `AMQPError` due to subsequent API calls via .e.g `basic_publish` or `queue_declare` within the exception handler. But in that case you would still see the warning "Reconnected to RabbitMQ because connection was closed" in the logs before the exception.

But imo the API router shouldn't be dealing with AMQP related stuff. As part of the separation of concerns principle, it's the EventManager that needs to be dealing with this technology details